dspace/config/submission-forms.xml

```diff
@@ -2127,7 +2127,7 @@
                     <dc-qualifier>isreplacedby</dc-qualifier>
                     <repeatable>true</repeatable>
                     <label>Is replaced by</label>
-                    <input-type>onebox</input-type>
+                    <input-type autocomplete-custom="solr-handle_title_ac">autocomplete</input-type>
```

Comment:
rename to autocomplete-solr?

Reply:
no, because it is not only solr. Some suggestions could be also loaded from the *.json file.